assets/app/view/entity_order.rb

```diff
@@ -8,6 +8,7 @@ def render
       style = {
         display: 'inline-block',
         'margin-right': '1rem',
+        'white-space': 'nowrap'
```

Comment:
is the entity order scrollable now? i've been wanting to do that

Reply:
I'm not sure what you mean by scrollable; I just did this so that the browser wouldn't try to wrap at the space between the company acronym and the president.

Reply:
on an iphone, the entity order goes to multiple lines annoyingly, can you make it always one line and scroll on smaller phones, also centering the entity that's active?

Reply:
I don't know how to do that offhand. Maybe make that a new issue and see if someone with the right knowledge picks it up? I can also look into it eventually if no one grabs it.

Reply:
ok